draft-ietf-masque-connect-ip.md

```diff
@@ -835,7 +835,7 @@ In particular, they do not necessarily have IPv6 link-local addresses.
 Additionally, IPv6 stateless autoconfiguration or router advertisement messages
 are not used in such interfaces, and neither is neighbor discovery.
 
-Clients MAY optimistically start sending proxied IP packets before receiving
+When using HTTP/2 or later, a client MAY optimistically start sending proxied IP packets before receiving
```

Comment:
Wondering if "later" is the correct thing here. Perhaps:
```suggestion
When using HTTP/2 or HTTP/3, or future HTTP versions, a client MAY optimistically start sending proxied IP packets before receiving
```

Reply:
This sounds like we are introducing a design contract that applies to all the future versions of HTTP. Are we fine with that?

Reply:
Agreed, switched to h2 and h3